apiary.apib

```diff
@@ -3464,6 +3466,28 @@ Update a task.
 
 + Response 204
 
+### tasks.complete [POST /tasks.complete]
+
+Mark a task as complete.
```

Comment:
```suggestion
Mark a task as completed.
```

Reply:
What is the proper spelling in this case actually?

Reply:
I think they can both be used.

Actually, thinking a bit more about this, we pretty much always use the term *done* instead of complete. Would it make sense to call the endpoint `/tasks.done`? Or I guess in that case it would need to be `/tasks.markAsDone`? 

I used `/tasks.complete` because that's what we have in api-future, but we don't really use that term anywhere.

Reply:
^ I guess if we would do this we would also need to rename the property on the task from `complete` to `done`.